pages/view.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div>
+    <v-container>
+      <v-row>
+        <iframe
+          class="col-12 height-70vh"
+          :src="$route.query.url"
+          frameborder="0"
+          @error="handleError"
+        ></iframe>
+      </v-row>
+    </v-container>
+    <v-container class="p-4">
+      <p>
+        If above iFrame is not working you can click on the following button to
+        view on actual domain.
+      </p>
+      <v-row class="d-flex" justify="center">
+        <a :href="$route.query.url" class="d-block">
+          <v-chip class="ma-2" color="primary" style="cursor: pointer">
+            Full content
+          </v-chip>
+        </a>
+      </v-row>
+    </v-container>
+  </div>
```

Comment:
change the logic of this page and show a detail page.

there won't be an API that will be used for fetching the details, figure out how from the listing page you can navigate to the details page and show the article, if you have any doubts understanding the requirement please connect.

Reply:
It's possible to render the details page if user open the page using provided link as state will be available on that time, but if it refresh the page then only solution is to store that particular news on local storage and use as data are changing continuously.

Reply:
please implement the functionality with option 1

Reply:
resolved, if user refresh on details page it redirect to home page